src/box/sql/select.c

```diff
@@ -2188,22 +2192,25 @@ sqlColumnsFromExprList(Parse * parse, ExprList * expr_list,
 				if (zName[j] == '_')
 					nName = j;
 			}
-			zName =
-			    sqlMPrintf(db, "%.*z_%u", nName, zName, ++cnt);
+			zName = sqlMPrintf(db, "%.*z_%u", nName, zName, ++cnt);
```

Comment:
Just curious: doesn't it leak memory allocated for `zName` with `sqlDbStrDup` above?

Reply:
No, these values are added to a hash table and removed during cleanup below.